Makefile

```diff
@@ -13,7 +15,8 @@ TEA_BUILD_BENCHMARKS ?= OFF
 TEA_BENCHMARK_REPETITIONS ?= 10
 # Note: If the tea benchmark format or output is changed, make changes to ./benchmark/runall.sh
 TEA_BENCHMARK_FORMAT ?= json
-TEA_BENCHMARK_OUTPUT ?= $(PROJECT_ROOT)/tea/benchmarks/reports/tea-bench-results.$(TEA_BENCHMARK_FORMAT)
+TEA_BENCHMARK_OUTPUT ?= $(PROJECT_ROOT)/tea/benchmarks/reports/tracer-tea-bench-results.$(TEA_BENCHMARK_FORMAT)
+BENCHMARK_EXTRA ?=
```

Comment:
What are we using `BENCHMARK_EXTRA` for?

Reply:
Here you can see https://github.com/DataDog/dd-trace-php/pull/2791/files#diff-c4ae11bb97d3556005b5a12e3b22a451f2586dc0fc49f340e864a2ee6ee1b07dR42 
Basically, I don't want to run all tests for appsec just framework ones and this will allow me to do so